src/mumble/AudioOutput.cpp

```diff
@@ -128,8 +133,17 @@ float AudioOutput::calcGain(float dotproduct, float distance) {
 }
 
 void AudioOutput::wipe() {
-	foreach (AudioOutputUser *aop, qmOutputs)
-		removeBuffer(aop);
+	QList< AudioOutputBuffer * > list;
+
+	qrwlOutputs.lockForRead();
```

Comment:
This should use a lock-guard instead of manual locking (and potentially a dedicated scope for restricting its lifetime)